src/App.vue

```diff
@@ -248,10 +245,14 @@ export default class App extends Mixins(BcolMixin, DateMixin, FilingTemplateMixi
     return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
   }
 
-  /** Handle State Changes */
-  private stateChangeHandler (hasStateChanges: boolean): void {
-    this.showFeeSummary = hasStateChanges
-    this.setHaveCorrection(hasStateChanges)
+  private get showFeeSummary (): boolean {
+    const defaultFilingData = {
+      filingTypeCode: null,
+      entityType: null,
+      priority: false,
+      waiveFees: false
+    }
+    return this.isFilingChanged && !this.isSame(this.getFilingData, defaultFilingData)
```

Comment:
I don't understand why you need to check for different filing data...?

Reply:
because the isfilingChanged returns even before filing data get set. Looking at the code I dont understand why that happens. When that happens Fee Summary component errors out. So as a safety check I added this

Reply:
I think the store does think async (ie, in another tick) which is why things don't seem to get set right away.

Please add a comment explaining this safety check.

Reply:
Done